db/migrate/20150401061243_create_notifications.rb

```diff
@@ -0,0 +1,18 @@
+class CreateNotifications < ActiveRecord::Migration
+  def change
+    create_table :notifications do |t|
+      t.integer :course_id, null: false, foreign_key: { references: :courses }
+      t.integer :user_id, null: false, foreign_key: { references: :users }
+      t.string :action
+      t.string :activity_content
```

Comment:
What I thought is that most activity feeds are also notifications to users. For example, students may level up or gain achievements frequently. All those could be both feeds and notifications. So I put them together